examples/user_guide/12-Responding_to_Events.ipynb

```diff
@@ -344,26 +344,79 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from holoviews.streams import Params\n",
+    "class BankAccount(param.Parameterized):\n",
+    "    balance = param.Number(default=0, doc=\"Bank balance in USD\")\n",
+    "    overdraft = param.Number(default=200, doc=\"Overdraft limit\")\n",
     "\n",
-    "class Style(param.Parameterized):\n",
-    "\n",
-    "    cmap = param.ObjectSelector(default='viridis', objects=['viridis', 'plasma', 'magma'])\n",
+    "account = BankAccount(name='Jane', balance=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can link parameter changes straight to DynamicMap callable parameters using the streams dictionary format (HoloViews version >= 1.14.2):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "streams = dict(total=account.param.balance, overdraft=account.param.overdraft, owner=account.param.name)\n",
     "\n",
-    "    color_levels = param.Integer(default=255, bounds=(1, 255))\n",
+    "def table(owner, total, overdraft):\n",
+    "    return hv.Table([(owner, overdraft, total)], ['Owner', 'Overdraft ($)', 'Total ($)'])\n",
     "\n",
-    "style = Style()\n",
+    "bank_dmap = hv.DynamicMap(table, streams=streams)\n",
+    "bank_dmap.opts(height=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now is you set the `balance` parameter on the `janes_account` insteance, the DynamicMap above updates. Note that the dictionary specifies that the `balance` parameter is mapped to the `total` argument of the callable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "account.balance=65.4\n",
+    "account.overdraft=350"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Use with `panel`\n",
     "\n",
-    "stream = Params(style)\n",
+    "This dictionary format is particularly useful when used with the [Panel](http://panel.pyviz.org/) library (a dependency of HoloViews that should always be available) as `panel` widgets always reflect their values on the `value` parameter. This means that if you declare two panel widgets as follows:"
```

Comment:
```suggestion
    "This dictionary format is particularly useful when used with the [Panel](http://panel.pyviz.org/) library (a dependency of HoloViews that should always be available), because `panel` widgets always reflect their values on the `value` parameter. This means that if you declare two Panel widgets as follows:"
```